components/address/AddressCardInner.vue

```diff
@@ -0,0 +1,74 @@
+<script setup lang="ts">
+import type { Schemas } from '@shopware/api-client/api-types';
+import { getTranslatedProperty } from '@shopware/helpers';
+
+withDefaults(
+    defineProps<{
+      headline?: string;
+      address?: Schemas['CustomerAddress'],
+      description?: string;
+    }>(),
+    {
+        headline: undefined,
+        address: undefined,
+        description: undefined,
+    },
+);
+
+const configStore = useConfigStore();
+const showTitle = configStore.get('core.loginRegistration.showTitleField') as boolean;
+const showAdditionalAddressField1 = configStore.get('core.loginRegistration.showAdditionalAddressField1') as boolean;
+const showAdditionalAddressField2 = configStore.get('core.loginRegistration.showAdditionalAddressField2') as boolean;
+const showPhoneNumberField = configStore.get('core.loginRegistration.showPhoneNumberField') as boolean;
+</script>
+
+<template>
+    <div>
+        <slot name="address-card-headline">
+            <h3 v-if="headline" class="mb-2 border-b border-gray-100 pb-2 text-lg font-bold">
```

Comment:
```suggestion
            <h3 v-if="headline" class="mb-2 border-b border-gray-100 pb-2">
```

i add general styling for the h3 in the useStyle pr. maybe we can remove it here to have identical h3 everwhere

Reply:
done